tests/Functional/WebhookControllerTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Tests\Funcitonal;
+
+use Tests\HttpTestCase;
+use Illuminate\Http\Response;
+use Tests\Handlers\ValidHandler;
+
+class WebhookControllerTest extends HttpTestCase
+{
+    /**
+     * @test
+     */
+    public function it_should_return_200()
+    {
+        $data = [
+            'event-type' => 'user.updated',
+            'data' => [
+                'id' => 1
+            ]
+        ];
+
+        $res = $this->signedJson('POST', '/webhook', $data, [], 'webhooks');
+
+        $this->assertSame(Response::HTTP_OK, $res->getStatusCode());
```

Comment:
The response object has a nice helper for this, `$res->assertOk()`. Also, `$res->assertStatus(...)` for other response status codes.